src/DucksNet.API/Controllers/TreatmentController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DucksNet.Domain.Model;
+using DucksNet.Infrastructure.Prelude;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DucksNet.API.Controllers;
+
+[ApiController]
+[Route("api/v1/[controller]")]
+public class TreatmentController : ControllerBase
+{
+    private readonly IRepository<Treatment> _treatmentRepository;
+    public TreatmentController(IRepository<Treatment> repository)
+    {
+        _treatmentRepository = repository;
+    }
+
+    [HttpGet]
+    public IActionResult GetAll()
+    {
+        var treatments = _treatmentRepository.GetAll();
+        return Ok(treatments);
+    }
+
+    [HttpGet("{ownerId}")]
```

Comment:
Swagger can't make sense of these because there are multiple Get routes with the same shape. Make it `user/{userId}` etc...